homeassistant/components/sensor/tellduslive.py

```diff
@@ -146,4 +146,4 @@ def device_class(self):
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return "-".join(self._id[0:2])
+        return "-".join(map(str, self._id))
```

Comment:
```suggestion
        return "{}-{}".format(*self._id)
```

Reply:
Python is so cool but this would be better as the fix is due to the length of `self._id` is 3.

`return "{}-{}-{}".format(*self._id)`